teams/mods.toml

```diff
@@ -9,6 +9,9 @@ members = [
     "matthieu-m",
 ]
 
+[[github]]
```

Comment:
On GitHub the team is currently called `moderation`, so we either add `team-name = "moderation"` or we rename the existing team.

Reply:
I'd rather rename the github team, I'd prefer to have everything consistent with the names used in the team repo. Ultimately having two teams is also not the end of the world.

Reply:
Ok, renamed the team in the GitHub UI! Resolving this comment.